shared/src/hover/HoverOverlay.scss

```
@@ -82,16 +82,25 @@
         // stylelint-enable selector-max-compound-selectors
     }
 
+    &__alerts {
+        display: flex;
+        flex-direction: column;
+    }
     &__alert {
-        padding: 0.5rem;
+        padding: 0.25rem 0.5rem;
         margin: 0;
         border-radius: 0;
         border: none;
         border-bottom: 1px solid var(--border-color);
+
+        p {
+            display: inline;
+        }
     }
 
     &__alert-close {
         float: right;
+        margin-left: 0.75em;
```

Probably meant `rem`?
```suggestion
        margin-left: 0.75rem;
```